### DISCUSSION TITLE
Block Theme Resource Round-Up

### DISCUSSION THREAD

**Discussion by DaisyOlsen** [WordPress/developer-blog-content#45] [Topic Ideas]
Similar to the topic idea in #37 - I'd like to write up a round-up style post with resources for Theme Developers, particularly for Block Themes and the gradual adoption of block theme technology. Ideally, focused on content that was new in the last several months. Since the post will not be published until 2023, I suggest avoiding calling it a 2022 wrap-up in favor of a more general Round-up.

- workshops/courses/tutorials on learn.wordpress.org
- wordpress.tv videos
- dev notes
- Useful how-to/tutorials from DevHub

To be published before the end of January 2023.

Discussion, feedback welcome!

**Comment by bph** [comment]
Published: [Block theme resources roundup](https://developer.wordpress.org/news/2023/02/block-theme-resources-roundup/)
